src/repl/scala/tools/nsc/interpreter/IMain.scala

```diff
@@ -865,7 +865,7 @@ class IMain(initialSettings: Settings, protected val out: JPrintWriter) extends
         |${preambleHeader format lineRep.readName}
         |${envLines mkString ("  ", ";\n  ", ";\n")}
         |$importsPreamble
-        |${indentCode(toCompute)}""".stripMargin
+        |%s""".stripMargin.format(indentCode(toCompute))
```

Comment:
That goes back to retronym completion. I'd have to investigate. I see it changed in my change you reverted? Presumably it was off by one at some point. https://github.com/scala/scala/commit/f279894a3efe84b85e9becfd0ce96aaa0c21bfbd

It should really import int + or - => offWhichWay, then `len offWhichWay 1`.